docs/configuration/downstream/pull_from_upstream.md

```diff
@@ -7,7 +7,10 @@ sidebar_position: 1
 
 A dist-git only job that opens a new dist-git pull request in
 src.fedoraproject.org when a new upstream release happens using a notification
-from [release-monitoring.org](https://release-monitoring.org/).
+from [release-monitoring.org](https://release-monitoring.org/). Upstream Release Monitoring creates a Bugzilla
+for the new upstream release and this Bugzilla is by default referenced in the dist-git changelog as
+`- Resolves rhbz#xz` or in the commit message in case of using `%autochangelog`. This behaviour can be customized
```

Comment:
Actually, don't we add this to the commit message even if not using `%autochangelog`?

Reply:
no, we check whether the specfile uses %autochangelog... but we could still change this before releasing, WDYT

Reply:
after discussing with Franta, the behaviour was changed and therefore the docs updated as well